lte/gateway/c/core/oai/tasks/amf/amf_config.c

```diff
@@ -0,0 +1,584 @@
+  ngap_conf->port_number            = NGAP_PORT_NUMBER;
+  ngap_conf->outcome_drop_timer_sec = NGAP_OUTCOME_TIMER_DEFAULT;
+}
+
+/***************************************************************************
+**                                                                        **
+** Name:    amf_config_init()                                             **
+**                                                                        **
+** Description: Initializes default values for AMF                        **
+**                                                                        **
+**                                                                        **
+***************************************************************************/
+void amf_config_init(amf_config_t* config) {
+  memset(config, 0, sizeof(*config));
+
+  pthread_rwlock_init(&config->rw_lock, NULL);
+
+  config->max_gnbs                       = 2;
+  config->max_ues                        = 2;
+  config->unauthenticated_imsi_supported = 0;
+  config->relative_capacity              = RELATIVE_CAPACITY;
+  config->amf_statistic_timer            = AMF_STATISTIC_TIMER_S;
+  config->use_stateless                  = false;
+  ngap_config_init(&config->ngap_config);
+  nas5g_config_init(&config->nas_config);
+  guamfi_config_init(&config->guamfi);
+  m5g_served_tai_config_init(&config->served_tai);
+}
+
+/***************************************************************************
+**                                                                        **
+** Name:    amf_config_parse_opt_line()                                   **
+**                                                                        **
+** Description: Invokes amf_config_init() to initialize                   **
+**              default values of AMF                                     **
+**                                                                        **
+**                                                                        **
+***************************************************************************/
+int amf_config_parse_opt_line(int argc, char* argv[], amf_config_t* config_pP) {
+  amf_config_init(config_pP);
+  return 0;
+}
+
+/***************************************************************************
+**                                                                        **
+** Name:    amf_config_parse_opt_line()                                   **
+**                                                                        **
+** Description: Invokes amf_config_init() to initialize                   **
+**              default values of AMF                                     **
+**                                                                        **
+**                                                                        **
+***************************************************************************/
+int amf_config_parse_file(amf_config_t* config_pP) {
```

Comment:
For shared config between mme and amf, rather than parsing them again, why not copy from the mme config?

Reply:
Separate Zenhub ticket(#8332) opened to handle